index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nicolas Olaya Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+    <div class="wrapped">
+        <nav class="navbar">
+            <h2>Welcome</h2>
+            <a href="#" class="options"><i class="las la-bars"></i></a>
+            <a href="#" class="shortcuts">Portfolio</a>
+            <a href="#" class="shortcuts">About</a>
+            <a href="#" class="shortcuts">Contact</a>
```

Comment:
To write semantic code, when you see that there is a list of links or items in the HTML code, you can use `ul` tag with `li` elements.
Then place these `a` tags inside each `li` tag.